pandas/tests/extension/base/base.py

```diff
@@ -1,17 +1,2 @@
-import pandas._testing as tm
-
-
 class BaseExtensionTests:
-    # classmethod and different signature is needed
-    # to make inheritance compliant with mypy
-    @classmethod
-    def assert_equal(cls, left, right, **kwargs):
-        return tm.assert_equal(left, right, **kwargs)
-
-    @classmethod
-    def assert_series_equal(cls, left, right, *args, **kwargs):
-        return tm.assert_series_equal(left, right, *args, **kwargs)
-
-    @classmethod
-    def assert_frame_equal(cls, left, right, *args, **kwargs):
-        return tm.assert_frame_equal(left, right, *args, **kwargs)
+    pass
```

Comment:
This removal is not backward compatible for extensionarrays tests that inherit BaseExtensionTests. IMO this should be deprecated instead?

Reply:
Is there a downstream EA that is actually affected or is this just in principal?

If the latter, I'm inclined to be more aggressive with changing the tests than the non-test code.  I don't want to get in the business of writing deprecation tests for the tests.

Reply:
Every downstream EA that reuses those tests can be affected if they use the same pattern in those classes (eg in a test they override). See the two linked PRs below for two such examples. 

Now, I think the question is: is there any downstream EA implementation that actually makes _use_ of those test class methods (i.e. by overriding them, and thus not relying purely on the base `tm.assert_.._equal` functions). That I don't know (the two examples of affected projects don't do this, and so for them it's an easy fix)